### ISSUE TITLE
Organization and Documentation of the code 

### ISSUE THREAD

**Issue by adrishyantee** [DSCKGEC/Resumie#31]
We need proper documentation and organization of the code so that it can be easily understood by newcomers. Make sure that the tasks are executed within functions and comment on how they are working, This is quite a tedious thing to do as you have to go through the whole codebase line by line, but you will learn a lot through this process.

**Comment by Akshit6828** [comment]
Hi did you mean we need to add comments in the code for understanding the code? Please correct me if I am wrong!

**Comment by adrishyantee** [comment]
> Hi did you mean we need to add comments in the code for understanding the code? Please correct me if I am wrong!


Yeah, you are right! Adding comments in and around functions, using proper variable names, and whatsoever makes the code look neat.
